http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoMetricsMonitor.scala

```diff
@@ -441,6 +449,11 @@ class LeoMetricsMonitor[F[_]](config: LeoMetricsMonitorConfig,
         )
       })
       .getOrElse(List.empty)
+
+  private def isUpgradeable(appType: AppType, cloudProvider: CloudProvider, chart: Chart): Boolean =
```

Comment:
This was an idea of mine… the current apps dashboard is fairly cluttered, since there are a bunch of old apps in the system which won't be upgraded (some of which are failing health checks, etc). I'm adding an `upgradeable` label to app metrics, which should let us filter those old apps out of the dashboard.

Reply:
Nice!